docs/architecture/decisions/0003-handover-from-sirius.md

```diff
@@ -0,0 +1,34 @@
+# 3. Handover from Sirius
+
+Date: 2024-03-14
+
+## Status
+
+Accepted
+
+## Context
+
+Users will access the Paper ID service from Sirius, starting from an LPA's details page. They will select which person and cases are being ID-checked and then be taken to the Paper ID service.
+
+We need a way to pass the person and case data between services.
+
+## Decision
+
+Sirius will generate a URL of the Paper ID service including the type of person being ID-checked and cases that the check should impact. It will then redirect the user to that link.
+
+The structure of the URL will be `/start?personType={type}&lpas[]={uid}`. `{type}` will be either "donor" or "certificate-provider". `{uid}` will be the case's UID; multiple `{uid}` parameters can be provided.
```

Comment:
Assume the multiple uids are provided using PHP style array capture from query params? i.e. `&lpas[]={uid1}&lpas[]={uid12}` if so might want to be specific on that or people might try comma separate list etc..

Reply:
Good point, I've added bulked out the "multiple parameters" bit into a separate paragraph/sentence.